packages/foreman/nodejs-axios/nodejs-axios.spec

```diff
@@ -1,19 +1,17 @@
 %global npm_name axios
-%global enable_tests 1
 
-Name: nodejs-%{npm_name}
-Version: 0.17.1
+Name: nodejs-axios
+Version: 0.19.0
 Release: 1%{?dist}
 Summary: Promise based HTTP client for the browser and node
 License: MIT
 Group: Development/Libraries
 URL: https://github.com/axios/axios
 Source0: https://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
 BuildRequires: nodejs-packaging
-Requires: npm(follow-redirects) >= 1.2.5
-Requires: npm(follow-redirects) < 2.0.0
-Requires: npm(is-buffer) >= 1.1.5
-Requires: npm(is-buffer) < 2.0.0
+Requires: npm(follow-redirects) = 1.5.10
```

Comment:
This is packaged at 1.4.1 so needs a bump as well. Can safely be done because nothing else depends on it

Reply:
should this and the one below be bundled or an update for each one ?

Reply:
I would prefer to keep it unbundled for now.